src/ImageSharp/Primitives/Rational.cs

```diff
@@ -74,6 +74,11 @@ public Rational(double value, bool bestPrecision)
 
         this.Numerator = (uint)rational.Numerator;
         this.Denominator = (uint)rational.Denominator;
+        
+        if(this.Numerator == 0 && this.Denominator == 0) 
```

Comment:
If there's an issue here it will have to be fixed in [`LongRational`](https://github.com/SixLabors/ImageSharp/blob/7d0bbbdf9388009c22ad0382a4d49ca8bdb700c0/src/ImageSharp/Primitives/LongRational.cs#L132)

Reply:
if you want to update GPS location of ExifInformation, it accepts Retional[]. That's why I suggest this change. Anyway if you are creating Rational.FromDouble(0) Denominator shouldn't be 0.

Reply:
That's the correct fix. Can you also add some unit tests for this to make sure this stays fixed?

Reply:
yes, sure will do that and will push all as a different pull request. You can close this one.

Reply:
No need to open a new PR. That’s just confusing. Push to this one please.

Reply:
And you can also use a force push to update this PR with a new set of commits.